### PR TITLE
Update exploit-protection-exploit-guard.md

### PR DESCRIPTION
**Pull request by Reeced40**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/3622
Added a review section.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@levinec A new PR that addresses issue #3622 is available for review. Thanks.

File: windows/security/threat-protection/windows-defender-exploit-guard/exploit-protection-exploit-guard.md [hunk 1]
```diff
@@ -45,6 +45,19 @@ Many of the features in the [Enhanced Mitigation Experience Toolkit (EMET)](http
 >[!WARNING]
 >Some security mitigation technologies may have compatibility issues with some applications. You should test exploit protection in all target use scenarios by using [audit mode](audit-windows-defender-exploit-guard.md) before deploying the configuration across a production environment or the rest of your network.
 
+## Review exploit protection events in the Windows Defender ATP Security Center
+
+Windows Defender ATP provides detailed reporting into events and blocks as part of its alert investigation scenarios.
+
+You can query Microsoft Defender ATP data by using [Advanced hunting](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/advanced-hunting-windows-defender-advanced-threat-protection). If you're using [audit mode](audit-windows-defender-exploit-guard.md), you can use Advanced hunting to see how exploit protection settings would affect your environment if they were enabled.
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-exploit-guard/exploit-protection-exploit-guard.md]:
```suggestion
You can query Microsoft Defender ATP data by using [Advanced hunting](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/advanced-hunting-windows-defender-advanced-threat-protection). If you're using [audit mode](audit-windows-defender-exploit-guard.md), you can use Advanced hunting to see how exploit protection settings could affect your environment.
```

File: windows/security/threat-protection/windows-defender-exploit-guard/exploit-protection-exploit-guard.md [hunk 2]
```diff
@@ -45,6 +45,19 @@ Many of the features in the [Enhanced Mitigation Experience Toolkit (EMET)](http
 >[!WARNING]
 >Some security mitigation technologies may have compatibility issues with some applications. You should test exploit protection in all target use scenarios by using [audit mode](audit-windows-defender-exploit-guard.md) before deploying the configuration across a production environment or the rest of your network.
 
+## Review exploit protection events in the Windows Defender ATP Security Center
```
Comment (levinec) [inline windows/security/threat-protection/windows-defender-exploit-guard/exploit-protection-exploit-guard.md]: update to "Microsoft Security Center"

File: windows/security/threat-protection/windows-defender-exploit-guard/exploit-protection-exploit-guard.md [hunk 3]
```diff
@@ -45,6 +45,19 @@ Many of the features in the [Enhanced Mitigation Experience Toolkit (EMET)](http
...
+
+Windows Defender ATP provides detailed reporting into events and blocks as part of its alert investigation scenarios.
```
Comment (levinec) [inline windows/security/threat-protection/windows-defender-exploit-guard/exploit-protection-exploit-guard.md]: update to "Microsoft Defender ATP"

**Comment by e0i** [comment]
@Reeced40 Please follow-up with suggested changes and tag me whenever it is ready. Thanks.

**Comment by Reeced40** [comment]
@e0i done thanks Onur.

**Pull request merged**